src/components/InstanceMonitoring/_instance-monitoring.scss

```diff
@@ -0,0 +1,51 @@
+@import 'carbon-components/scss/globals/scss/colors';
+@import 'carbon-components/scss/globals/scss/typography';
+@import 'carbon-components/scss/globals/scss/vars';
+
+.instance-card {
+  background-color: #fff;
+  width: 1167px;
+  height: 210px;
+  .instance-metrics {
+    display: flex;
+    justify-content: center;
+    li {
+      padding: 20px;
```

Comment:
Does it make sense to use `rem()` function in `carbon-components/scss/globals/scss/typography` for those geometry stuffs, too?